src/server/license/db.go

```diff
@@ -0,0 +1,22 @@
+package license
+
+import (
+	"github.com/jmoiron/sqlx"
+	"golang.org/x/net/context"
+)
+
+// CreateClustersTable sets up the postgres table which tracks active clusters
+func CreateClustersTable(ctx context.Context, tx *sqlx.Tx) error {
+	_, err := tx.ExecContext(ctx, `
+CREATE TABLE IF NOT EXISTS license.clusters (
```

Comment:
I was surprised when I realized that this service interacts directly with postgres for clusters (vs using the collections library, which does point at etcd now but which @Tryneus is porting to postgres). I think the only other places that use postgres directly right now are a few low-level PFS libraries in `src/internal/storage` (and FWIW the collections library does do the proto serialization/deserialization for you, though it looks like `gogoproto.moretags`, which was cool to learn about, does a lot of lifting as well).

I think it might make sense to double-check with JD or something about how it important it is to have everything go through the collections library (unless this just can't, in which case, that answers that. I think maybe this is a piece of code I don't totally understand, since this uses the collections library for the license itself). Anyway, if he thinks it's fine to have different code work differently, then it seems fine to me.

Reply:
Yeah we'll need to have the discussion about whether the collections library makes sense for postgres. My 2-cents is that using a RDBMS as a key-value store loses a lot of value. For example, I want to be able to get a count of all the clusters for testing license constraints, which should be fast in postgres but slow in etcd. We're also not going to serialize data in postgres as protobufs (I hope). It's not really clear how one should write models at the moment. `gogoproto.moretags` is pretty convenient if the model looks a lot like the response, but that's not always the case.

Reply:
Update: Per slack conversation, it sounds like we're fine with two storage interfaces, so this seems fine (there was some concern about shared/overlapping transactions, but since the license service isn't necessarily adjacent to any other pachd server, and therefore may not share storage with any other pachd API, I don't think shared transactions are generally achievable anyway).